client/state/reader/thumbnails/test/actions.js

```diff
@@ -81,7 +81,7 @@ describe( 'actions', () => {
 						thumbnailUrl,
 					} );
 
-					expect( dispatchSpy.calledTwice );
```

Comment:
Either more actions were added and the test didn't fail or the test was copypasta that didn't fail in the first instance.